src/internal/pfsdb/model.go

```diff
@@ -84,11 +85,24 @@ func parseBranches(branchInfos string) ([]*pfs.Branch, error) {
 	return branches, nil
 }
 
-// Commit is a row in the pfs.commits table.
 type Commit struct {
-	ID          CommitID `db:"id"`
-	Repo        Repo     `db:"repo"`
-	CommitSetID string   `db:"commit_set_id"`
+	RowID          CommitID      `db:"int_id"`
+	RepoID         RepoID        `db:"repo_id"`
+	CommitSetID    string        `db:"commit_set_id"`
+	CommitID       string        `db:"commit_id"`
+	BranchID       sql.NullInt64 `db:"branch_id"`
```

Comment:
Lets group branch related info under the comment `// BranchName is used to derive the BranchID in commit related queries.`